nav2_msgs/srv/IsPathValid.srv

```diff
@@ -1,6 +1,7 @@
 #Determine if the current path is still valid
 
 nav_msgs/Path path
+uint8 max_cost
```

Comment:
You can set the default value to `254`, so that if not populated, the behavior doesn't change. 

```
fieldtype fieldname fielddefaultvalue
```